change fastmode to fastscan in dev-guide (#9937)
pingcap/docs: sql-statements/sql-statement-set-tiflash-mode.md

> **Warning:**
>

> > This statement is still an experimental feature. 

Reviewer: ```suggestion
> This statement is still an experimental feature. It is NOT recommended that you use it in the production environment.
```

> The form and usage of related experimental features may change in subsequent versions.

You can use the `ALTER TABLE...SET TIFLASH MODE...` statement to switch the FastScan option for the corresponding table in TiFlash. The following options are currently supported.

> - `Normal Mode`. The default option, means disabling FastScan. In this option, TiFlash guarantees the accuracy of query results and data consistency.
> - `Fast Mode`. The FastScan option, means enableing FastScan. In this option, TiFlash provides more efficient query performance, but does not guarantee the accuracy of query results and data consistency.

Reviewer 2: ```suggestion
- `Normal Mode`. The default option. In this mode, FastScan is disabled and the accuracy of query results and data consistency are guaranteed.
- `Fast Mode`. In this mode, FastScan is enabled. TiFlash provides more efficient query performance, but does not guarantee the accuracy of query results and data consistency.
```

> This statement executes without blocking the execution of existing SQL statements or the running of TiDB features, such as transactions, DDL, and GC, and without changing the data content accessed through the SQL statement. The statement will end normally when the option switch is completed.

Reviewer 2: ```suggestion
The execution of `ALTER TABLE ... SET TIFLASH MODE ...` does not block existing SQL statements or interrupt other TiDB features, such as transactions, DDL, and GC. At the same time, data accessed through SQL statements is not altered. This statement ends normally when the mode is switched.
```

> This statement only supports switching the FastScan option for tables in TiFlash, so the switch only affects reads involving the TiFlash table.

Reviewer 2: ```suggestion
You can only execute this statement to switch the FastScan option for tables in TiFlash. Therefore, option change only affects reading tables in TiFlash.
```

> The FastScan switch takes effect only if the table has a TiFlash Replica. If the TiFlash Replica of the table is empty when you switch the option, the option will take effect only after the TiFlash Replica is subsequently reset. You can use [`ALTER TABLE ... SET TIFLASH REPLICA ...`](/sql-statements/sql-statement-alter-table.md) to reset the TiFlash Replica.

Reviewer 2: ```suggestion
The FastScan switch takes effect only if the table has TiFlash replicas. If no TiFlash replica is present when you switch the option, the option takes effect only after TiFlash replicas are configured. You can use [`ALTER TABLE ... SET TIFLASH REPLICA ...`](/sql-statements/sql-statement-alter-table.md) to configure TiFlash replicas.
```

> **Warning:**
>

> > This statement is still an experimental feature. It is NOT recommended that you use it in the production environment.

Reviewer: ```suggestion
> This statement is still an experimental feature.
```

+------------+
```

> If you want to enable FastScan to query the `test` table, execute the following statement to switch the option, and you can query the option of the current table.

Reviewer 2: ```suggestion
To enable FastScan for the `test` table, execute the following statement and then check whether FastScan is enabled for this table.
```

+------------+
```

> If you want to disable FastScan, execute the following statement to switch.

Reviewer 2: ```suggestion
To disable FastScan, execute the following statement.
```
